helix-rest/src/main/java/org/apache/helix/rest/server/resources/zookeeper/ZooKeeperAccessor.java

```diff
@@ -165,6 +179,24 @@ private Response getChildren(BaseDataAccessor<byte[]> zkBaseDataAccessor, String
     }
   }
 
+  /**
+   * Returns the ZNode Stat object given the path.
+   * @param zkBaseDataAccessor
+   * @param path
+   * @return
+   */
+  private Response getStat(BaseDataAccessor<byte[]> zkBaseDataAccessor, String path) {
+    if (zkBaseDataAccessor.exists(path, AccessOption.PERSISTENT)) {
+      Stat stat = zkBaseDataAccessor.getStat(path, AccessOption.PERSISTENT);
```

Comment:
I think getStat return null if the path does not exist. So you don't need to call exists before this line.

Reply:
Changed